orchagent/routeorch.cpp

```diff
@@ -1951,3 +2005,767 @@ bool RouteOrch::removeOverlayNextHops(sai_object_id_t vrf_id, const NextHopGroup
     return true;
 }
 
+void RouteOrch::doLabelTask(Consumer& consumer)
```

Comment:
This is considerable code to routeorch. Suggest moving to a different file `mplsrouteorch.cpp`, @abdosi ?

Reply:
The functionality to process LABEL_ROUTE_TABLE and ROUTE_TABLE is almost identical in nature.  Both process NH/NHG association with label or prefix.  Moving to a separate file file implies that the functionality is inherently different, which isn't true.

Reply:
I agree but at the same time why do you need a separate task for label route? Other than SAI API what is different.

Reply:
I have moved the MPLS related functions into a separate file:  routeorch_mpls.cpp

Reply:
lets not have `_` in filename. Suggest follow existing naming and rename to `mplsrouteorch.cpp`

Reply:
@prsunny I disagree.  I want the file name to make it clear that it contains the MPLS related functions for RouteOrch, not MplsRouteOrch.  I looked at other file naming in orchagent and _ is present in other filename, so there is precedent.

Reply:
`_ `is not currently present for orch files and prefer not to have different naming conventions. IMO, mplsrouteorch.cpp is clear.

Reply:
_ is not present in other orch files because no other Orch component is split into multiple files due to overlarge file size.

Reply:
Still it doesn't mean we need to have a different new naming convention.

Reply:
I still see this open. Please address this.

Reply:
I have checked the file lengths in sonic-swss/orchagent and found that there are three other Orch components with file sizes larger than routeorch.cpp.  Subdividing RouteOrch for file size reasons will not be done within the context of this PR.

Reply:
Sorry, I dont prefer mpls changes to be present as part of routeorch.

Reply:
@prsunny Done